scripts/check.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+import argparse
+import sys
+from logging import INFO, basicConfig, getLogger
+
+from bugbug.models.component import ComponentModel
+
+basicConfig(level=INFO)
+logger = getLogger(__name__)
+
+
+class ModelChecker:
+    def go(self, model_name):
+        # TODO: Stop hard-coding them
+        valid_models = ["component"]
```

Comment:
Maybe we should have a module in bugbug that defines all model names and their classes?

Reply:
Any other idea?

Reply:
Having a central places where the models are defined and the list of possible model names seems a good idea. We can start by manually maintaining the list and see if we need to dynamically builds it later. Do you want me to do it in a separate PR or this one?

Reply:
Yeah, a separate one. Maybe we can merge that one first, since this one depends on the creation of the new images on Docker Hub.

Reply:
Filed #367 for this.